web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -175,3 +175,28 @@ textarea
     &:before
       +icon-size(15px)
       content: '\e617'
+
+// Additional styling for dropbutton.
+.dropbutton-action
```

Comment:
Honestly, the styles here don't look too consistent to the other elements for few reasons:
- this element looks like a button:
![btn1](https://user-images.githubusercontent.com/1923476/28580253-3f834a48-715f-11e7-930e-5b82661177c3.png)
However, it's not clickable on the whole surface and it's just a span - tricky both for accessibility and usability
- the colors are very different than this button which is just next to the button dropdown:
![btn2](https://user-images.githubusercontent.com/1923476/28580319-6c7dd18a-715f-11e7-8d82-96c7ff6f96ee.png)

In comparison of those 2, the first one looks disabled.

In addition, some attributes for accessibility for this type of element are missing:
https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20161214/#menubutton
https://w3c.github.io/aria-practices/#menubutton)
Examples:

https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20161214/examples/menu-button/menu-button-1/menu-button-1.html
https://w3c.github.io/aria-practices/examples/menu-button/menu-button-links.html